app/src/test/java/net/aiscope/gdd_app/presentation/SelectDiseasePresenterTest.kt

```diff
@@ -40,7 +41,15 @@ class SelectDiseasePresenterTest {
     @Test
     fun `should store a disease with a name in the repository`() =
         coroutinesTestRule.runBlockingTest {
-            whenever(repository.create()).thenReturn(Sample("id", "hospital", "a microscopist"))
+            whenever(repository.create(anyString())).thenReturn(
```

Comment:
Why not using `DISEASE_NAME` instead of `anyString()`? It would make the test more specific.

Reply:
Sure